content/source/docs/enterprise/saml/identity-provider-configuration-aad.html.md

```diff
@@ -7,53 +7,109 @@ page_title: "SAML Azure Active Directory Identity Provider Configuration - Terra
 
 Follow these steps to configure Azure Active Directory (AAD) as the identity provider (IdP) for Terraform Enterprise.
 
--> **Note:** This guide assumes you have an appropriate licensing agreement for Azure Active Directory that supports non-gallery application single sign-on.
+~> **Note:** This guide assumes you have an appropriate licensing agreement for Azure Active Directory that supports non-gallery application single sign-on.
```

Comment:
```suggestion
-> **Tip:** > We suggest you use the [documentation](https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/terraform-enterprise-tutorial) Microsoft provides regarding setting up the TFE SSO gallery application.

~> **Note:** This guide assumes you have an appropriate licensing agreement for Azure Active Directory that supports non-gallery application single sign-on.
```

Reply:
Can we just add this to be right after the intro sentence at the top? a note and a tip right in a row can be visually overwhelming for readers :)

Reply:
Also - are the instructions we link to duplicating any of the content below? If so, I wonder if we should remove it, link to the tutorial for those steps, and then have sections on this page that users should do after they have followed that process. Thoughts?